rosetta/admin.py

```diff
@@ -0,0 +1,63 @@
+from functools import update_wrapper
+
+from django.contrib import admin
+
+from rosetta.signals import reload_server
+
+try:
+    from django.urls import reverse_lazy
+except:
+    from django.core.urlresolvers import reverse_lazy
+from django.views.generic import RedirectView
+
+from rosetta.models import TranslationBackup, Translation
+from rosetta.conf import settings as rosetta_settings
+
+
+def restore(modeladmin, request, queryset):
+    for backup in queryset:
+        backup.restore()
+
+    if rosetta_settings.AUTO_RELOAD:
+        reload_server.send(sender=None, request=request)
```

Comment:
Usually, good practice to add smth as sender
https://docs.djangoproject.com/en/1.11/topics/signals/#connecting-to-specific-signals